website/docs/r/app_service_environment_v3.html.markdown

```diff
@@ -111,13 +111,11 @@ A `cluster_setting` block supports the following:
 
 ## Attribute Reference
 
-* `id` - The ID of the App Service Environment.
+In addition to the Arguments above, the following Attributes are exported:
```

Comment:
Could you leave the ID in, it's always exported on resources.

Reply:
done!